src/core/config/mod.rs

```diff
@@ -1250,6 +1250,17 @@ pub struct Config {
 	#[serde(default = "default_ip_range_denylist")]
 	pub ip_range_denylist: Vec<String>,
 
+	/// Optional interface to bind to with SO_BINDTODEVICE for URL previews.
+	/// If not set, it will not bind to a specific interface.
+	/// This uses [`reqwest::ClientBuilder::interface`] under the hood.
+	///
+	/// To list the interfaces on your system, use the command `ip link show`
+	///
+	/// Example: `"eth0"`
+	#[cfg(any(target_os = "android", target_os = "fuchsia", target_os = "linux"))]
+	#[serde(default)]
```

Comment:
For `Option` config options, don't specify a serde default. Otherwise, this gets translated by our config generator to `false`, and because it doesn't need it anyways.

Reply:
Make sure you regenerate the config